### ISSUE TITLE
Don't ask before building

### ISSUE THREAD

**Issue by flyrev** [ariya/phantomjs#13479]
I think it is really unnecessary to ask before building PhantomJS.

The warning is fine, but asking is a bit too much.

**Comment by ariya** [comment]
In the ideal world, yes this is not necessary. However, since the message was introduced, there was a significant drop in the support activities, e.g. explaining to others why building it from source is not for everyone. I don't know have any quantatitive data as to how many hours of my time (and other's) have been saved by this simple message, but I can assure you that every little thing adds up.

In other words, unless someone can assure me that more folks are actually reading the documentation, I am storngly in favor of keeping the message.

**Comment by flyrev** [comment]
My change removes the question, not the message. I see your point though.

**Comment by zackw** [comment]
Without the question, the message will be lost behind the flood of compilation messages and the user won't get a chance to read it.  My grey Unix beard doesn't like it either, but I think it's necessary.

You can use `./build.sh --confirm` on the command line instead of answering the question every time.
